### ISSUE TITLE
Selbstständigkeitserklärung bei unterschiedlichen Optionen für Abschlussarbeiten

### ISSUE THREAD

**Issue by Yasiminala** [tudace/tuda_latex_templates#320]
Der Text in der Selbstständigkeitserklärung lautet immer 
"Hiermit versichere ich, NAME, _die_ vorliegende **type** ohne Hilfe Dritter ..."

Ist es möglich für einen selbstgewählten **type** oder für ein Projekt-Proposal das "die" in ein "das" zu ändern?

Außerdem wird in der Selbstständigkeitserklärung im dritten und vierten Absatz immer von einer Thesis gesprochen. Soll dies auch für ein Projekt-Proposal der Fall sein?

**Comment by TeXhackse** [comment]
Das ist teilweise richtig. Der Text ist leider exakt in der Form für Abschlussarbeiten vorgegeben, ich wäre daher vorsichtig mit anpassungen, da ich nicht garantieren kann, dass das für andere Dokumententypen als Abschlussarbeiten übernommen werden kann. 

Allerdings könnte ich das zusammen mit einer entsprechenden Warnung versehen, dass diesbezüglich keine Garantie besteht. 
@schoeps @marklaz  was meint ihr dazu?

**Comment by schoeps** [comment]
Ich würde ungerne Varianten in die Welt setzen, die nicht rechtssicher sind. Kann man nicht schon jetzt einfach einen komplett eigenen Text an diese Stelle setzen?  Dann kann jede/r sowieso genau das schreiben was er/sie mag (aber eben auf eigenes Risiko und ohne trügerische Sicherheit).

**Comment by TeXhackse** [comment]
Ja, komplett ersetzen geht.

**Comment by LordMaddhi** [comment]
Gibt es eine Möglichkeit die digitale Variante ohne das "Bei der abgegebenen Thesis stimmen die schriftliche und [...] elektronische Fassung [...] überein" zu nutzen? Gibt es einen Command dafür?

**Comment by TeXhackse** [comment]
Eine Option wäre noch die Textbausteine einteln per Makro aufrufbar zu machen, dann können Nutzer:innen sich daraus eine eigene Zusammenbauen, und wir lassen die fest so wie sie ist.

**Comment by schoeps** [comment]
@Yasiminala war Ihnen bewusst, dass man den Text einfach komplett ersetzen kann? Reicht Ihnen das?

**Comment by Yasiminala** [comment]
Vielen Dank für die Antworten!
Ansich fände ich eine automatische (sprachlich korrekte) Anpassung des Artikels - zumindest für die vorgegebenen types - optimal. Ich ging allerdings nicht davon aus, dass es rechtlich ein Problem darstellen könnte, den Artikel anzupassen.

Ich werde mal in meiner Arbeitsgruppe nachfragen, wie das bei uns gehandhabt wird und den Text dann gegebenenfalls durch die Änderung vom "die" in "das" komplett ersetzen. 
Könnten Sie mir vielleicht noch sagen, wie der entsprechende Befehl für das komplette Ersetzen lautet?

**Comment by schoeps** [comment]
Sorry, ich sehe jetzt, dass ich die Anfrage falsch verstanden hatte und erst jetzt begreife. Ich dachte, dass man für "type" nur zwischen Bachelor und Master wählen kann, aber man kann hier komplett eigene Strings benutzen? Und @Yasiminala's AG nutzt das auch für Projektarbeiten etc?

**Comment by TeXhackse** [comment]
ja, der type wird automatisch übernommmen und eben sprachlich nicht angepasst. der rest-text soll anscheinend gleich bleiben.

**Comment by schoeps** [comment]
@Yasiminala Eigener Text geht so (steht auch in der [Doku](http://mirror.ctan.org/macros/latex/contrib/tuda-ci/doc/DEMO-TUDaThesis.pdf)) 
```
\begin{affidavit*}[Babel-Sprachoption]{Überschrift}
Text
\end{affidavit*}
```

**Comment by schoeps** [comment]
>ja, der type wird automatisch übernommen und eben sprachlich nicht angepasst. der rest-text soll anscheinend gleich bleiben.

Bei vordefinierten Typen (z.B. `pp`) könnten wir ja die Artikel tatsächlich leicht intern abspeichern und dann passend ändern und auch "Thesis" durch "Proposal" ersetzen, oder? Das finde ich nachvollziehbar und sehe da jetzt auch keine große  Rechtsunsicherheit? Für eigene Typen erscheint mir das aber nicht praktikabel, denn da dann müsste man ja auch die Artikel abfragen... das erscheint mir overkill.

**Comment by TeXhackse** [comment]
Wenn der Artikel nicht statisch ist, ist der eh ein abhängig Makro, dieses Makro könnnten Nutzer:innen dann auch überschreiben. Es wäre dann also ohnehin egal, die Frage ist nur wie viel man davon offiziell supporten und dokumentieren will.

**Comment by schoeps** [comment]
> Wenn der Artikel nicht statisch ist, ist der eh ein abhängig Makro, dieses Makro könnnten Nutzer:innen dann auch überschreiben. Es wäre dann also ohnehin egal, die Frage ist nur wie viel man davon offiziell supporten und dokumentieren will.

Ich würde vorschlagen das mit dem Artikel als internes Makro zu lösen und nicht zu dokumentieren. So kann dann auch niemand meckern, wenn es das später nicht mehr funktioniert :)

**Comment by TeXhackse** [comment]
Okay, mach ich so.
